web/src/composebox_typeahead.ts

```diff
@@ -1351,13 +1446,16 @@ export function initialize({
             }
             return sorted;
         },
+
```

Comment:
Why are you adding spaces here in an inconsistent fashion? I don't see how it's an improvement.